.gitignore

```diff
@@ -5,3 +5,4 @@ npm-debug.log
 yarn-error.log
 screenshots
 .idea
+package-lock.json
```

Comment:
I didn't add it - npm added it itself at some point. I'll remove the ignore and we can make a ticket about including it.